KrysiaBartek2024/source/toc_entries.tex

```diff
@@ -0,0 +1,1105 @@
+\section{\texorpdfstring{1\mins\ opening}{1mOpening}}\label{sec:1mOpening}
+
+\subsubsection*{1\clubs\ -- ?}
+\begin{itemize}
+    \item 1\diams\ = 0-6
+    \item 1\hearts\ = 4+\hearts
+    \item 1\spades\ = 4+\spades
+    \item 1\ntx\ = 7-10, no 4\majs
+    \item 2\clubs\ = 12-14 BAL or \clubs, \gf
+    \item 2\diams\ = no 4\majs, 5+\diams, \gf
+    \item 2\hearts\ = 5\spades4\hearts\ 6-9
+    \item 2\spades\ = 11+ BAL, no 4\majs
+    \item 2\ntx\ = 11-12 BAL
+    \item 3\nt\ = 15-17 BAL
+\end{itemize}
+
+\subsubsection*{1\diams\ -- ?}
+\begin{itemize}
+    \item 1\hearts\ = 4+\hearts
+    \item 1\spades\ = 4+\spades
+    \item 1\ntx\ = 6-10, no 4\majs, no \diams\ support
+    \item 2\clubs\ = 12-14 BAL or \clubs, \gf
+    \item 2\diams\ = no 4\majs, 4+\diams, \gf
+    \item 2\hearts\ = 5\spades4\hearts\ 6-9
+    \item 2\spades\ = 11+ BAL, no 4\majs
+    \item 2\ntx\ = 11-12 BAL
+    \item 3\nt\ = 15-17 BAL
+\end{itemize}
+
+\subsubsection*{1\clubs\ -- 2\clubs \\ ?}
+\begin{itemize}
+    \item 2\diams\ = BAL
+    \item 2\hearts\ = 5\clubs4\hearts\ UNBAL
```

Comment:
UNBAL robi imo syf

Reply:
to dałam unbalanced ale teraz się imo gryzie z BAL

Reply:
tak może? 
![image](https://github.com/OilyCannelloni/macaronibridge/assets/71830127/abac87c4-2e78-4195-abd8-62d3bbc58e15)